SPM/.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
+/.build
+/Packages
+xcuserdata/
+DerivedData/
```

Comment:
I did't have the perfect place to ask for it: edit README as well cause there is still instructions to use Cocoapods. It would be good to have instruction related to SPM instead.